core/src/test/scala/zio/jdbc/SqlFragmentSpec.scala

```diff
@@ -159,6 +159,13 @@ object SqlFragmentSpec extends ZIOSpecDefault {
                 assertIn(List(1, 2, 3)) &&
                 assertIn(Vector(1, 2, 3)) &&
                 assertIn(Set(1, 2, 3))
+              } + test("multiple interpolation params") {
```

Comment:
Maybe worth testing with more than 2? Didn't read the implementation but 2 items can often be managed in a way that 3+ items wouldn't work.

Reply:
No. It's not necessary. The root cause of the issue was in the parameter indexing of the `ZConnection.executeSqlWith` function. Now, it covers sequences correctly, and quantity doesn't matter.

Reply:
My point is that it's not necessary with the current implementation but a future change could make it necessary to detect a bug. Up to you but I prefer to always use 3+ items in such tests.

Reply:
I added more items to the existing test and added one more to cover the empty collection.